wit_dashboard/wit-inference-panel.html

```diff
@@ -486,6 +426,22 @@
         'updateModelSignature(modelSignature1, modelSignature2)',
       ],
 
+      async ready() {
+        // Early exit if this is called in environment outside of Tensorboard.
+        if (!tb_plugin_lib) {
+          return;
+        }
+        // Get URL params for the WIT plugin and set this element's properties
+        // based on those params.
+        const urlParams =
+          await tb_plugin_lib.experimental.core.getURLPluginData();
+        const paramList = Object.keys(urlParams);
+        for (let i = 0; i < paramList.length; i++) {
+          const param = paramList[i];
+          this.set(param, urlParams[param]);
```

Comment:
With tf.storage we had the ability to both set the url if user changes a variable within WIT and set the variables if user set the url outside of WIT. This code only works URL-> WIT direction. It would be good to add a TODO here to close the WIT->URL loop in future when TB supports it.

Reply:
done